components/tokens/custom-express/custom-large-vars.css

```diff
@@ -15,4 +15,7 @@ governing permissions and limitations under the License.
 .spectrum--express.spectrum--large {
   --spectrum-colorwheel-path: "M 118 118 m -118 0 a 118 118 0 1 0 236 0 a 118 118 0 1 0 -236 0.2 M 118 118 m -92 0 a 92 92 0 1 0 184 0 a 92 92 0 1 0 -184 0";
   --spectrum-colorwheel-path-borders: "M 120 120 m -120 0 a 120 120 0 1 0 240 0 a 120 120 0 1 0 -240 0.2 M 120 120 m -90 0 a 90 90 0 1 0 180 0 a 90 90 0 1 0 -180 0";
+  --spectrum-colorloupe-express-loupe: url(#express-loupe-clip-l);
+  --spectrum-colorloupe-express-m-border: none;
+  --spectrum-colorloupe-express-l-border: block;
```

Comment:
can I have the custom tokens in this PR or do I need to put them in separate PR like we did for ColorWheel.

Reply:
Oh, I've no idea. If Patrick had it done in a separate PR then I guess, let's do that again just to be safe?